src/app/pages/faq/faq.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SharedModule } from '@shared/shared.module';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { FaqComponent } from './faq.component';
+
+const COMPONENTS = [FaqComponent];
+
+@NgModule({
+  declarations: [...COMPONENTS],
+  imports: [CommonModule, SharedModule, FlexLayoutModule, ReactiveFormsModule, MatInputModule],
```

Comment:
All the modules are imported via shared, can be removed :+1:
```suggestion
  imports: [SharedModule],
```

Reply:
Yeah, those modules shouldn't even be imported in faqModule anyway. My mistake, for copy-pasting xD

Reply:
Fixed in 32a3833